relay/nakama/signer/signer.go

```diff
@@ -0,0 +1,20 @@
+package signer
+
+// signer.go manages the creation and loading of the Nakama private key used to sign
+// all transactions.
+
+import (
+	"context"
+
+	"pkg.world.dev/world-engine/sign"
+)
+
+type Signer interface {
+	SignTx(ctx context.Context, personaTag string, namespace string, data any) (*sign.Transaction, error)
+	SignSystemTx(ctx context.Context, namespace string, data any) (*sign.Transaction, error)
+	SignerAddress() string
```

Comment:
is the `SignerAddress` just the public key? maybe this method should be called `PublicKey` instead

Reply:
It's not just the public key; It's generated using [crypto.PubkeyToAddress](https://pkg.go.dev/github.com/ethereum/go-ethereum/crypto#PubkeyToAddress)